ttnn/cpp/ttnn/operations/normalization.hpp

```diff
@@ -184,6 +184,9 @@ struct GroupNorm {
         const std::optional<ttnn::DataType> dtype = std::nullopt,
         std::optional<CoreGrid> core_grid = std::nullopt,
         std::optional<bool> inplace = std::nullopt) {
+        if (input_tensor.get_layout() == Layout::TILE) {
```

Comment:
Please use assert. User asked for in_place, but this check silently ignores it which can later lead to a fairly nasty debugging session for a user.

Something like
> assert (input_tensor layout is tile and inplace is not requested) - inplace is not supported for tiled inputs because ...

Reply:
does assert error out? Do we have a warning for it